### PR TITLE
Update the notes around the TOS changes in july 2017

### PR DESCRIPTION
**Pull request by orta**
This changes the wording on the from-page around our ToS changes in #119

### CODE REVIEW

File: app/views/welcome/index.html.haml [hunk 1]
```diff
@@ -17,12 +17,14 @@
 
 .alert.alert-danger
   .p
-    Note that the API is in super early beta and is currently available for internal, experimental use only.
+    Note that the API is in beta, and is subject to change. Howeverm available for non-commercial production use.
```
Comment (dblock) [inline app/views/welcome/index.html.haml]: Why is the API still beta? I think we should say that the API is ready for production use, fix the typo in "however", or just remove it.

**Comment by orta** [comment]
Agreed, I've simplified it 👍

**Comment by anandaroop** [comment]
Just noting that the Google Groups welcome message reflected the older legal status so I changed it to match the [new wording](https://github.com/artsy/doppler/pull/124/files#diff-9cf437d888780ea267171393ac00ea3eR20):

after

**Comment by dblock** [comment]
Much thanks @anandaroop!

**Pull request merged**